ios/Open In Thunder/Resources/manifest.json

```diff
@@ -0,0 +1,95 @@
+{
+    "manifest_version": 3,
+    "default_locale": "en",
+
+    "name": "__MSG_extension_name__",
+    "description": "__MSG_extension_description__",
+    "version": "1.0",
+
+    "icons": {
+        "48": "images/icon-48.png",
+        "96": "images/icon-96.png",
+        "128": "images/icon-128.png",
+        "256": "images/icon-256.png",
+        "512": "images/icon-512.png"
+    },
+
+    "content_scripts": [{
+        "js": [ "content.js" ],
+        "matches": [
+            "*://aussie.zone/*",
```

Comment:
I'm guessing this probably can't be moved to its own file. If not, we can just have the workflow recreate this whole thing. 😊

Reply:
Yeah, this is similar to the other comment - I don't think there's a way to import things into JSON files from other files but I might be wrong